validator/testcases/javascript/traverser.py

```diff
@@ -15,6 +15,7 @@
 IGNORE_POLLUTION = False
 POLLUTION_COMPONENTS_PATH = re.compile(r"/?components/.*\.jsm?")
 POLLUTION_EXCEPTIONS = set(["Cc", "Ci", "Cu", ])
+MAX_LINE_COMPLEXITY = 100
```

Comment:
That looks like a lot. I was expecting something like 10. Are you afraid of false positives?